course3/homework-string/test.cc

```diff
@@ -65,14 +66,25 @@ void string_test()
 	// operator +=
 	plusResult += string2;
 	assert(plusResult == "Hex123Hex");
+
+	// move contructor and assignment operator tests
+	StringType string4 = "Note to SSO: I am not a short string, so please don't try any fancy stuff!";
```

Comment:
Unit tests now pass for std::string as well (using clang 7). I quite enjoyed this hack, and learned a few things too :)

See previous comment here:
https://github.com/maciekgajewski/c-excercises/pull/26/commits/41be1231be54003e2aacf5a946798fcf1ffc0074